### PR TITLE
feat(dashboards): Issue widget count tooltips clickable and redirect to discover

### PR DESCRIPTION
**Pull request by edwardgou-sentry**
Makes the count tooltip for issue widgets clickable and redirects the user to discover
image

### CODE REVIEW

File: static/app/components/stream/group.tsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ import {
   Query,
 } from 'sentry/views/issueList/utils';
 
-const DiscoveryExclusionFields: string[] = [
+export const DiscoveryExclusionFields: string[] = [
```
Comment (evanpurkhiser) [inline static/app/components/stream/group.tsx:52]: Should we move this into a constants file?

Also should it be called `DISCOVER_EXCLUSION_FIELDS` ?

Comment (edwardgou-sentry) [reply static/app/components/stream/group.tsx]: 👍  moved and renamed!

File: static/app/utils/dashboards/issueFieldRenderers.tsx [hunk 2]
```diff
@@ -174,6 +182,24 @@ const issuesCountRenderer = (
...
+) => {
+  const commonQuery = {projects: [Number(data.projectId)]};
+  const discoverView = EventView.fromSavedQuery({
+    ...commonQuery,
+    id: undefined,
+    name: data.title,
+    fields: ['title', 'release', 'environment', 'user', 'timestamp'],
+    orderby: '-timestamp',
+    query: `issue.id:${data.id}${filtered ? data.discoverSearchQuery : ''}`,
+    version: 2,
```
Comment (wmak) [inline static/app/utils/dashboards/issueFieldRenderers.tsx:198]: I'm seeing a discrepancy here, if my dashboard has a date range (eg. 24h) when i click this link that date range isn't carried over (eg. discover is at 14d) which means the counts don't match.

**Review by wmak — APPROVED** [review]
Minor bug with statsPeriod, otherwise LGTM

**Pull request merged**